Core/BFFT/frutil/Ticks.hpp

```diff
@@ -8,31 +8,22 @@
 #ifndef BFFT_FRUTIL_TICKS_HPP_
 #define BFFT_FRUTIL_TICKS_HPP_
 
+#include "FreeRTOS.h"
 
+
+/**
+ * @fn Ticks
+ * @brief Convert diverse units to ticks
+ */
 class Ticks
 {
 public:
-	static const Ticks Max_Delay;
-
-	const TickType_t ticks;
-
-	Ticks(TickType_t ticks): ticks(ticks)
-	{}
+	const TickType_t cxTicks;
+	static const Ticks cMaxDelay;
 
-	static Ticks fromTicks(TickType_t ticks)
-	{
-		return Ticks(ticks);
-	}
-
-	static Ticks fromMs(unsigned long ms)
-	{
-		// TODO: check for overflow
-		return Ticks(portTICK_RATE_MS * ms);
```

Comment:
In my opinion a multiplication is wrong, we should devide by portTick_RATE_MS instead.
Please correct me if I am wrong :)

Reply:
Yes you are right @rkeller-1